### ISSUE TITLE
Oxygen auto filling <div> and replacing stuff created inside a code-block

### ISSUE THREAD

**Issue by runechronos** [soflyy/oxygen-bugs-and-features#2273]
To see the weird behavior :

1/ Go to https://lescarlinottes.com/location-meribel-en-ete/

2/ Scroll a little bit and Find this `<div>` in the inspector:
`<div class="embed" data-embed-app="filesapp/index.php" data-embed-path="lescarlinottes">login</div>`

3/ Inside the code block i created in the builder, I only created an empty `<div>` like this:
`<div class="embed" data-embed-app="filesapp/index.php" data-embed-path="lescarlinottes"></div>`

--> The problem is Oxygen filled the word "login" inside the div I don't know why, preventing what i'm trying to use from a 3rd party script to execute properly (it is supposed to show a dynamic gallery not the word "login")

--> The `<div>` needs to remain empty in my case or else, it can't show the gallery from my 3rd party script, what should I do to prevent any automatic behavior from Oxygen in specific content like this `<div>` ?

In summary, I would like to know if it's possible in Oxygen, to tell Oxygen : don't do anything on this element, just let it render on his own (i thought it would be ok with a code block)

Of course, i deactivated every other plugins except oxygen during my tests and cleared all caches multiple times

Thanks in advance for your help, it is crucial for my business
